backend/tracim_backend/lib/utils/authentication.py

```diff
@@ -32,41 +44,121 @@
 AUTH_TOKEN_QUERY_PARAMETER = "access_token"
 CLIENT_TOKEN_HEADER = "X-Tracim-ClientToken"
 
+SAML_IDP_DEFAULT_CONFIG = {
+    "default": {
+        "logo_url": "",
+        "displayed_name": "",
+        "attribute_map": {
+            "user_id": "${mail}",
```

Comment:
I'm not sure to understand. I'm pretty sure in tracim user_id is an integer. This can't work

Reply:
I messed up the naming. User_Id actually binds to tracim's username. I will make it clearer with https://github.com/tracim/tracim/issues/6258

Reply:
If so @Millefeuille42 please add a checkbox in #6258 description so we are sure not to forget it.

Reply:
Will do

Reply:
done in https://github.com/tracim/tracim/pull/6259